code/components/handling-loader-five/src/HandlingLoader.cpp

```diff
@@ -79,7 +79,20 @@ static float getFloatField(char* handlingChar, uint32_t offset, const char* fiel
 {
 	auto hash = HashRageString(fieldName);
 
-	if (hash == HashRageString("fDriveBiasFront") || hash == HashRageString("fBrakeBiasFront") || hash == HashRageString("fSuspensionBiasFront") ||
+	if (hash == HashRageString("fDriveBiasFront"))
+	{
+		float fDriveBiasFront = *(float*)(handlingChar + offset);
+
+		if (fDriveBiasFront + *(float*)(handlingChar + offset + 4) > 1.0f)
```

Comment:
This could be instead
```c
float fDriveBiasRear = *(float*)(handlingChar + offset + 4);
if ((fDriveBiasFront == 1.0f && fDriveBiasRear == 0.0f) || (fDriveBiasFront == 0.0f && fDriveBiasRear == 1.0f))
{
	return fDriveBiasFront;
}
else
{
	return fDriveBiasFront / 2.0f;
}
```

Reply:
wow, reviewing your own code changes, cool!

why not force push that part instead

Reply:
kek done